src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -92,6 +96,25 @@ public bool ExecuteCommand(FormatDocumentCommandArgs args, CommandExecutionConte
                 c =>
                 {
                     var docOptions = document.GetOptionsAsync(c.CancellationToken).WaitAndGetResult(c.CancellationToken);
+                    var workspaceOptions = document.Project.Solution.Workspace.Options;
+
+                    // AB Test
+                    // If the feature is OFF and the user has not previously been assigned to a test group, assign them now
+                    if (!docOptions.GetOption(CodeCleanupOptions.PerformAdditionalCodeCleanupDuringFormatting)
+                        && workspaceOptions.GetOption(CodeCleanupABTestOptions.ParticipatedInExperiment) == false)
```

Comment:
> == false [](start = 104, length = 9)

Same comment as Heejae's above, you shouldn't need to compare with false, just use `!`.